pkg/kn/commands/humanreadable_flags.go

```diff
@@ -0,0 +1,54 @@
+// Copyright © 2019 The Knative Authors
```

Comment:
Can we make this file called human_readable_flags.go, or humanreadableflags.go?

Reply:
done